medusa/providers/torrent/html/morethantv.py

```diff
@@ -88,6 +88,9 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
         for mode in search_strings:
             logger.log('Search mode: {0}'.format(mode), logger.DEBUG)
 
+            if mode == 'Season':
+                search_strings[mode].append(re.sub(r'(.*)S0?', r'\1Season ', search_strings[mode][0]))
```

Comment:
problem of this is that if there is scene exceptions it will use only the first one.